### DISCUSSION TITLE
article/the-relationship-of-instruction

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#98] [Announcements]
# article/the-relationship-of-instruction

教等于学，学多于教，教大于学。强调学习是一个动态过程，学生既是接受者也是知识创造者。作者认为教是最好的学，通过教他人不仅促进别人学习，也强化了自己的知识体系。引用经典，作者提倡在学习中选择善者为师，同时倡导潜移默化的正面影响。最终，文章强调每个人都在不断影响世界，强调了教与学的相互关系。

https://blog.tangly1024.com/article/the-relationship-of-instruction

**Comment by happyzhangyyds** [comment]
很棒的想法，当时我在看费曼学习法的时候也是这种感受，但是如何权衡这其间的时间消耗则是更值得考量的问题，也涉及到双方交流的质量，如何更加善于提问和更加善于分享都是很好的方向。
